<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatosEmpleado</class>
 <widget class="QDialog" name="DatosEmpleado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/imagenes.qrc">
     <normaloff>:/save.jpeg</normaloff>:/save.jpeg</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/imagenes.qrc">
     <normaloff>:/atras.png</normaloff>:/atras.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3,1,6">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Empleado</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtEmpleado">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Categoría</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtCategoria">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4,1,4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ingreso     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtIngreso">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>N.I.F.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtNif">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4,1,4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Puesto      </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPuesto">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Nº Afiliación</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtNumeroAfiliacion">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4,1,4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Grupo Cotización </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtGrupoCotizacion">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Código Ocupación</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtCodigoOcupacion">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,8,1,4">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Contrato  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtContrato">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Finaliza Contrato</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtFinContrato"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imagenes/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
